### DISCUSSION TITLE
Updater & Vm

### DISCUSSION THREAD

**Discussion by FugginOld** [BassT23/Proxmox#117] [General]
Updater keeps getting an error and stops before going to the other nodes when it comes across my OPNsense VM. I have qemu and ssh enabled in the VM but still get a stop error 1. Anything I should look into?

**Comment by BassT23** [comment]
Please make an issue for that.

I see this also on my setup. The problem is: opnsense is an FreeBSD os.
FreeBSD is not implemented for now, so the  updater run into this error.
Hopefully I can fix it at this weekend ;)

For now, you can exclude this VM in config file.

In later release I think I also will make an config entry for FreeBSD, because, it could be critical to update the firewall.

**Comment by FugginOld** [comment]
Thank you. I wasn't sure if I was doing something wrong but I did end up adding them to the exclusion list.

I was seeing this issue with Linus distro VMs as well.

When you say please make an issue for that, do you mean with OPNsense?

**Reply by BassT23** [reply]
Make an issue here on this repo like "FreeBSD/Linus error"

THX

**Comment by BassT23** [comment]
beta v 4.1.7 fixed error.
will try to make free bsd updateable in next step ;)

**Comment by BassT23** [comment]
fixed in latest release.
FreeBSD skipped during update.
Will try to update FreeBSD in later release.
